fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -216,6 +216,8 @@ public static String valueList() {
 
     private PeriodDuration dataCachePartitionDuration;
 
+    private Boolean dataCacheEnable;
```

Comment:
looks like this variable is not used at all.

Reply:
The variables is used in the old version, I forgot to remove it, I'll remove it.